lib/Scheduler/hist_table.cpp

```diff
@@ -455,7 +456,8 @@ static bool doesHistorySLILCostDominate(InstCount OtherPrefixCost,
                                         LengthCostEnumerator *LCE) {
   auto RequiredImprovement = std::max(HistTotalCost - LCE->GetBestCost(), 0);
   auto ImprovementOnHistory = HistPrefixCost - OtherPrefixCost;
-  return ImprovementOnHistory <= RequiredImprovement;
+  // return ImprovementOnHistory <= RequiredImprovement;
+  return false;
```

Comment:
Why is this change correct? If it is, please delete the commented out code and the prior unused variables.

Reply:
Thank you for catching this